qutebrowser/utils/log.py

```diff
@@ -554,6 +580,9 @@ def format(self, record):
         record.__dict__.update(color_dict)
         return super().format(record)
 
+    def use_colors(self):
+        return self._use_colors
```

Comment:
No need to add a method here - just make `use_colors` "public" (i.e., remove the underline) and access it directly.